src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,60 @@
+package main.java.command;
+
+import main.java.components.Storage;
+import main.java.components.TaskList;
+import main.java.exception.InvalidInputException;
+
+/**
```

Comment:
Why is everything indented by 8 spaces?

I would prefer the indentation to be of 4 spaces to increase readability!

Reply:
How do I actually check the indentation though? It is 4 spaces in my Intellij and I have also set the settings in the editor accordingly.

Reply:
Hmmm... 

That's weird....
I would suggest opening it in visual studio code to double check the indentation! 

That's my suggestion...